Master

N03324541 opened the pull request.

N03324541 committed:
customization

N03324541 committed:
Blink output

N03324541 committed:
descrição

N03324541 committed:
Delete Cópia de newfile.txt

N03324541 committed:
assignment 2 - answers

N01371322 commented:
Please, no pull requests to me.
On Feb 21, 2016 8:10 AM, "N03324541" notifications@github.com wrote:

> ---
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/N01371322/ELSpring2016/pull/1
> Commit Summary
> - customization
> - Blink output
> - descrição
> - Delete Cópia de newfile.txt
> - assignment 2 - answers
> 
> File Changes
> - _A_ misc/Assignment2/Answers.txt
>   https://github.com/N01371322/ELSpring2016/pull/1/files#diff-0 (1344)
> - _A_ misc/Assignment2/Exercise_TerminalSession.txt
>   https://github.com/N01371322/ELSpring2016/pull/1/files#diff-1 (50)
> - _A_ misc/Screenshot 2016-02-02 às 17.59.52.png
>   https://github.com/N01371322/ELSpring2016/pull/1/files#diff-2 (0)
> - _A_ misc/newfile.txt
>   https://github.com/N01371322/ELSpring2016/pull/1/files#diff-3 (1)
> 
> Patch Links:
> - https://github.com/N01371322/ELSpring2016/pull/1.patch
> - https://github.com/N01371322/ELSpring2016/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/N01371322/ELSpring2016/pull/1.

N03324541 committed:
python code

N03324541 committed:
blinking LED

N03324541 committed:
blinking led

N03324541 committed:
LogTime

N03324541 committed:
log Temperature

This code gets the actual temperature and save it to the database with
the actual date and time.

N03324541 committed:
change the name of the file

N03324541 committed:
I added some comments to the functions and reorganized them. Now the
are easier to read.

N03324541 committed:
Schematic of the circuit

N03324541 committed:
Database used in the project

N03324541 committed:
Python code that controls the motor and the temperature sensor

N03324541 committed:
Python code that copies the database to a text file

N03324541 committed:
Shell script that execute both other python scrips and copies the
database to cloud server.
